### PR TITLE
openimageio, revbump, bump some dependency packages

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
`cmd:iconvert` is being checked by osl, hold of for now, maybe move cmd's back into base package.

File: media-libs/openimageio/patches/openimageio-2.3.10.0.patchset [hunk 1]
```diff
@@ -22,21 +22,6 @@ index a1ea8bd..9015ce7 100644
...
-diff --git a/src/libOpenImageIO/CMakeLists.txt b/src/libOpenImageIO/CMakeLists.txt
-index 699d592..059af0f 100644
---- a/src/libOpenImageIO/CMakeLists.txt
-+++ b/src/libOpenImageIO/CMakeLists.txt
-@@ -166,7 +166,9 @@ if (MINGW)
-     target_link_libraries (OpenImageIO PRIVATE ws2_32)
- endif()
- 
--
-+if (__HAIKU__)
```
Comment (korli) [inline media-libs/openimageio/patches/openimageio-2.3.10.0.patchset:34]: Shouldn't it be `if (HAIKU)`

Comment (Begasus) [reply media-libs/openimageio/patches/openimageio-2.3.10.0.patchset:34]: Nice catch, checking :)

Comment (Begasus) [reply media-libs/openimageio/patches/openimageio-2.3.10.0.patchset:34]: Needed PRIVATE too in there for conflicting target, thanks on the pointer!

**Comment by korli** [comment]
LGTM

**Pull request merged**